docker_start.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+
+[ -z "$CONTAINER_NAME" ] && CONTAINER_NAME=docker.io/Fe-Ti/iscraweb-ttbot:latest
```

Comment:
Maybe it's better to stop execution with error instead of hardcoding a link*:
```bash
[ -z "$CONTAINER_NAME" ] && die 'CONTAINER_NAME is not set! Set envvar $CONTAINER_NAME'
```

----
* Fun note: I've tried to create account with dash, but docker hub said that it's prohibited :-)